backend/EkoFunkcje/EkoFunkcje/Utils/InterventionFilterBuilder.cs

```diff
@@ -9,81 +9,115 @@ namespace EkoFunkcje.Utils
 {
     public static class InterventionFilterBuilder
     {
-        public static string GetInterventionGeoHashFilter(string geoHash, string interventionId)
+        public static string GetGeoHashFilter(string geoHash) {
+            return TableQuery.GenerateFilterCondition(
+                "PartitionKey", 
+                QueryComparisons.Equal,
+                geoHash
+            );
+        }
+
+        public static string GetByIdFilter(string interventionId)
         {
-            string geoHashFilter = TableQuery.GenerateFilterCondition(
-                "PartitionKey", QueryComparisons.Equal,
-                geoHash);
-            string idFilter = TableQuery.GenerateFilterCondition(
-                "RowKey", QueryComparisons.Equal,
-                interventionId);
+            return TableQuery.GenerateFilterCondition(
+                "RowKey",
+                QueryComparisons.Equal,
+                interventionId
+            );
+        }
 
-            string finalFilter = TableQuery.CombineFilters(
-                geoHashFilter,
-                TableOperators.And,
-                idFilter);
-            return finalFilter;
+        public static string GetInterventionGeoHashFilter(string geoHash, string interventionId)
+        {
+            return CombineFilters(
+                GetGeoHashFilter(geoHash), 
+                GetByIdFilter(interventionId)
+            );
         }
 
         public static string GetInterventionListViewFilter(ListInterventionsFilterRequest filter)
         {
-            string finalFilter = "";
+            List<string> filters = new List<string>();
             if (!string.IsNullOrWhiteSpace(filter.City))
             {
                 string cityFilter = TableQuery.GenerateFilterCondition(
                     "City", QueryComparisons.Equal,
                     filter.City);
-                finalFilter = TableQuery.CombineFilters(
-                    cityFilter,
-                    TableOperators.And,
-                    finalFilter);
+                filters.Add(cityFilter);
             }
             if (!string.IsNullOrWhiteSpace(filter.Street))
             {
                 string streetFilter = TableQuery.GenerateFilterCondition(
                     "Street", QueryComparisons.Equal,
```

Comment:
Jak możesz to przenieś te wszystkie "Street" do pól private static _streetFieldName = "Street"

Reply:
Stworzyłem prywatną statyczną klasę zamiast tego. Czy to jest ok?

Reply:
Mi się wydaje ok